pipeline_dp/dp_computations.py

```diff
@@ -104,3 +205,36 @@ def compute_dp_sum(sum: float, dp_params: MeanVarParams):
     return _add_random_noise(sum, dp_params.eps, dp_params.delta,
                              l0_sensitivity, linf_sensitivity,
                              dp_params.noise_kind)
+
+
+def compute_dp_mean(count: int, sum: float, dp_params: MeanVarParams):
+    """Computes DP mean.
+
+    Args:
+        count: Non-DP count.
+        sum: Non-DP sum.
+        dp_params: The parameters used at computing the noise.
+
+    Raises:
+        ValueError: The noise kind is invalid.
+    """
+    middle = dp_params.middle()
+    normalized_sum = sum - count * middle
+
+    l0_sensitivity = dp_params.l0_sensitivity()
+
+    # Splits the budget equally between the two mechanisms.
```

Comment:
@dvadym Is there a better way to split the budget between the mechanisms?

Reply:
For (eps, delta)-dp it's a good way. Mean computations for C++ DP library also uses such division. Later, there will be usage of PLD (Privacy Loss Distributions), it will be possible to split (eps, delta) in "more optimal" way.

Reply:
Sounds good, thanks!